packages/document/docs/en/plugin/official-plugins/last-updated.mdx

```diff
@@ -14,14 +16,7 @@ When you have configured `lastUpdated: true` in the default theme, this plugin w
 
 ## Installation
 
-```bash
-# npm
-npm install @rspress/plugin-last-updated
-# yarn
-yarn add @rspress/plugin-last-updated
-# pnpm
-pnpm install @rspress/plugin-last-updated
-```
+<PackageManagerTabs command="install @rspress/plugin-last-updated -D" />
```

Comment:
Can we just use `add` command? I think both npm and pnpm support `add` and it is shorter

Reply:
updated